doc/index.rst

```diff
@@ -10,6 +10,12 @@ documentation, respectively. In short, Hawkmoth is Sphinx Autodoc for C/C++.
 Hawkmoth aims to be a compelling alternative for documenting C and C++ projects
 using Sphinx, mainly through its simplicity of design, implementation and use.
 
+.. note::
+
+   The C++ support is still in early stages of development, and lacks some
+   fundamental features such as handling namespaces and documenting C++ specific
+   features other than classes.
```

Comment:
Not sure the part after 'and' is quite clear or accurate, though I don't think it needs to be in this case as it really is just a waiver.

For the record though, and while I haven't looked at this in some time, these are all the things I had on my todo list and at the time I felt confident there wouldn't be much more aside from the newest standards perhaps:

Important:
* namespace traversal
* using keyword

Not important:
* lambdas are treated as variables, not functions, though I'm not sure we want to go down that rabbit hole anyway, so much so that one could say the same about `std::function` objects or any user implementation of a similar thing.
* `friend` keyword, though it's an eye sore and no one should use it :shrug:

Reply:
> Not sure the part after 'and' is quite clear or accurate, though I don't think it needs to be in this case as it really is just a waiver.

Good enough to merge?

Reply:
Yeah, it's enough to scare most people away except those wanting to engage ;)